src/components/EllipsisLoader/index.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
please use standard test setup file: `import 'test-util/setup';`

no need to configure the Adapter each time: see line 3 and 6 - replace with the above line

Reply:
amended